addon/templates/components/polaris-avatar.hbs

```diff
@@ -1,29 +1,36 @@
-{{#if shouldShowInitials}}
-  <span class="Polaris-Avatar__Initials">
-    <svg class="Polaris-Avatar__Svg" viewBox="0 0 48 48">
-      <text
-        x="50%"
-        y="50%"
-        dy="0.35em"
-        fill="currentColor"
-        font-size="26"
-        text-anchor="middle"
-      >
-        {{initials}}
-      </text>
-    </svg>
-  </span>
-{{/if}}
+<span
+  role={{this.role}}
```

Comment:
Any reason to not just hard-code this now? 🤔

Reply:
ohh, it's private property
yeah, we can do 👍